src/components/Header/Header.tsx

```diff
@@ -18,6 +50,7 @@ const Header: FC<Props> = ({}) => {
         <Flex direction="row" alignItems="baseline" justifyContent="space-between">
             <h1>Signature Generator</h1>
             <Flex direction="row" alignItems="center" gap="size-100">
+                <RouteOptions />
```

Comment:
the new options get rendered here in the html though tis RouteOptions component JSX.